src/peer/negotiator.js

```diff
@@ -477,6 +438,77 @@ class Negotiator extends EventEmitter {
   }
 
   /**
+   * Replace the stream being sent with a new one.
```

Comment:
Can you add comments to differentiate `_replacePerTrack()` and `_replacePerStream()`?